LlmTornado/VectorStores/StaticChunkingConfig.cs

```diff
@@ -0,0 +1,26 @@
+using Newtonsoft.Json;
+
+namespace LlmTornado.VectorStores;
+
+/// <summary>
+/// Configuration options for static chunking
+/// </summary>
+public class StaticChunkingConfig
+{
+    /// <summary>
+    /// The maximum number of tokens in each chunk.
+    /// Default: 800
+    /// Minimum: 100
+    /// Maximum: 4096
+    /// </summary>
+    [JsonProperty("max_chunk_size_tokens")]
+    public int MaxChunkSizeTokens { get; set; }
```

Comment:
the properties being set to 0 implicitly and XML doc mentioning the default is 800/400 respectively could be confusing for the users and potentially rejected by the server (a reasonable expectation would be to either get null, then use the default or use & validate the value provided), so I'd set the default value here `{ get; set; } = 800;`